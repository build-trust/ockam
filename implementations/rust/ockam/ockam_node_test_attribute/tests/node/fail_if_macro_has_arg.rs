#[ockam_node_test_attribute::node(timeout = 100)]
async fn main(mut c: ockam::Context) {
    c.stop().await.unwrap();
}
