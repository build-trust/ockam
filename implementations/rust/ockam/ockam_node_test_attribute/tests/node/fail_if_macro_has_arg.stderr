error: This macro doesn't accept any argument
 --> $DIR/fail_if_macro_has_arg.rs:1:35
  |
1 | #[ockam_node_test_attribute::node(timeout = 100)]
  |                                   ^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/fail_if_macro_has_arg.rs:1:1
  |
1 | / #[ockam_node_test_attribute::node(timeout = 100)]
2 | | async fn main(mut c: ockam::Context) {
3 | |     c.stop().await.unwrap();
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/node/fail_if_macro_has_arg.rs`
